lxd/migrate.go

```diff
@@ -392,6 +392,15 @@ func (s *migrationSink) Connect(op *operations.Operation, r *http.Request, w htt
 		return err
 	}
 
+	// Set TCP timeout options.
+	remoteTCP, _ := tcp.ExtractConn(c.UnderlyingConn())
+	if remoteTCP != nil {
+		err = tcp.SetTimeouts(remoteTCP, 0)
+		if err != nil {
+			logger.Error("Failed setting TCP timeouts on remote connection", logger.Ctx{"err": err})
```

Comment:
If this is logged as an error, shouldn't it also return an error? If not, should we make it a warning instead?

Reply:
Yeah I agree it should be a  warning indeed. I dont want to fail migration though as its not really a big problem. I'll update the other side thus is set in too. Thanks

Reply:
Fixed.